### ISSUE TITLE
could not parse package version related with ubuntu

### ISSUE THREAD

**Issue by Hokwang** [quay/clair#802]
I installed clair latest version using harbor latest version.

very first log tells like below.
```
{"Event":"updater service started","Level":"info","Location":"updater.go:83","Time":"2019-06-19 09:44:27.761955","lock identifier":"dd57f7f5-02cd-45f5-bcfe-98f15da43c35"}
{"Event":"starting health API","Level":"info","Location":"api.go:85","Time":"2019-06-19 09:44:27.762006","port":6061}
{"Event":"sender configured","Level":"info","Location":"notifier.go:66","Time":"2019-06-19 09:44:27.762237","sender name":"webhook"}
{"Event":"notifier service started","Level":"info","Location":"notifier.go:82","Time":"2019-06-19 09:44:27.762345","lock identifier":"b5304f0e-0b75-4d25-b734-e432c20ce66f"}
{"Event":"attempting to obtain update lock","Level":"debug","Location":"updater.go:102","Time":"2019-06-19 09:44:27.764705"}
{"Event":"updating vulnerabilities","Level":"info","Location":"updater.go:192","Time":"2019-06-19 09:44:27.769509"}
{"Event":"fetching vulnerability updates","Level":"info","Location":"updater.go:239","Time":"2019-06-19 09:44:27.769547"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"oracle.go:119","Time":"2019-06-19 09:44:27.769614","package":"Oracle Linux"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"ubuntu.go:85","Time":"2019-06-19 09:44:27.769643","package":"Ubuntu"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"debian.go:63","Time":"2019-06-19 09:44:27.769733","package":"Debian"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"rhel.go:92","Time":"2019-06-19 09:44:27.769697","package":"RHEL"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"alpine.go:52","Time":"2019-06-19 09:44:27.769825","package":"Alpine"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:253","Time":"2019-06-19 09:44:29.680083","updater name":"alpine"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:253","Time":"2019-06-19 09:44:36.986577","updater name":"debian"}
{"Event":"could not parse package version. skipping","Level":"warning","Location":"ubuntu.go:316","Time":"2019-06-19 09:51:38.225981","error":"invalid version","version":"0.27-1+deb7u1build0.12.04.1, 0.28-1+deb8u1"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:253","Time":"2019-06-19 09:51:43.771029","updater name":"ubuntu"}
<snip>
{"Event":"fetcher note","Level":"warning","Location":"updater.go:214","Time":"2019-06-19 10:21:14.270355","note":"Ubuntu disco is not mapped to any version number (eg. trusty-\u003e14.04). Please update me."}
```

I think you have parsing problem about ubuntu.

Thanks.

### Environment
- Clair version/image: 2.0.8
- Clair client name/version: N/A
- Host OS: N/A
- Kernel (e.g. `uname -a`): N/A
- Kubernetes version (use `kubectl version`): N/A
- Helm version (use `helm version`): N/A
- Network/Firewall setup: N/A

**Comment by KeyboardNerd** [comment]
this `0.27-1+deb7u1build0.12.04.1, 0.28-1+deb8u1` is a known issue, we need to update the parser.
this `Ubuntu disco is not mapped to any version number` means we need to update the mapping from `disco` to whatever the canonical version is.

**Comment by Hokwang** [comment]
@KeyboardNerd 
Thanks for reply, first is known issue, do you have a plan for patch?
second is actually I can't understand. do you mean you will fix this problem or this is not a problem, the problem is in my configuration ?

**Comment by Hokwang** [comment]
@KeyboardNerd any updates?

**Comment by jzelinskie** [comment]
Yup this should have been fixed in 2.0.9 as per https://github.com/coreos/clair/commit/e4f71d5644adde0499ec3b2bbddee7303fbf7e07#diff-209d24465a2688f8893f3d383eb03851

**Comment by goidor** [comment]
Hello, I want to know If there is any workaround for this? since I still have that problem. Even with the latest version 2.1.4
